terminology/codesystem/GbaCodeSystemStorageTemperature.json

```diff
@@ -5,30 +5,33 @@
   "name": "GbaCodeSystemStorageTemperature",
   "title": "GbaCodeSystemStorageTemperature",
   "status": "draft",
-  "caseSensitive": true,
   "concept": [
     {
-      "code": "plus-2-to-plus-10",
+      "code": "temperature2to10",
```

Comment:
why the temperature prefix?

Reply:
Because these are the codes used in the BBMRI directory. Since the temperature categories are the same, i assumed the code system should be aligned with BBMRI.